.github/workflows/pull_request.yml

```diff
@@ -1,16 +1,16 @@
-name: Linting Python code
```

Comment:
In general, this whole commit is called `removed one line` but it looks like you changed something weird in each line of the code. But each line of the code looks the same. What is going on here?

Reply:
See this: https://github.com/remla23-team06/model-service/pull/4#issuecomment-1563986705

Reply:
@Iodine98 I guess that it doesn't introduce any meaningful changes, so I can drop it entirely right? 

The main problem is that when I'm trying to merge, there are a lot of conflicts that I have to resolve, since this commit introduces a formatting change at the end of each line of code.

Reply:
In general, I would suggest that you use one single IDE, whether that's PyCharm, VSCode or VIM so that you don't run into these problems.